### PR TITLE
[improvement](index)add index_id column in show index stmt

### PR DESCRIPTION
**Pull request by qidaye**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by qidaye** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowIndexStmt.java [hunk 1]
```diff
@@ -34,6 +34,7 @@ public class ShowIndexStmt extends ShowStmt {
     private static final ShowResultSetMetaData META_DATA =
             ShowResultSetMetaData.builder()
                     .addColumn(new Column("Table", ScalarType.createVarchar(64)))
+                    .addColumn(new Column("Index_id", ScalarType.createVarchar(30)))
```
Comment (xiaokang) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowIndexStmt.java:37]: add it to back

**Review by xiaokang — COMMENTED** [review]
change the testcase output

**Comment by qidaye** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**